cmd/fetch_repo/module.go

```diff
@@ -79,6 +79,7 @@ func fetchModule(dest, importpath, version, sum string) error {
 		cmd.Args = append(cmd.Args, "-modcacherw")
 	}
 	cmd.Args = append(cmd.Args, importpath+"@"+version)
+	cmd.Env = append(os.Environ(), "GO111MODULE=on")
```

Comment:
Let's move this to `internal/go_repository.bzl` next to where `GOSUMDB` is set to `off`. All the environment variable settings should be together.